components/admin/requests/doctor-information/EditModal.tsx

```diff
@@ -10,35 +10,30 @@ import {
   Text,
   Box,
 } from '@chakra-ui/react'; // Chakra UI
-import { useState, SyntheticEvent, ReactNode, useEffect } from 'react'; // React
+import { ReactNode } from 'react'; // React
 import DoctorInformationForm from '@components/admin/requests/doctor-information/Form';
 import { DoctorFormData } from '@tools/admin/requests/doctor-information'; // GraphQL types
+import { Form, Formik } from 'formik';
+import { editPhysicianInformationSchema } from '@lib/physicians/validation';
 
 type EditDoctorInformationModalProps = {
   children: ReactNode;
   readonly doctorInformation: DoctorFormData;
-  readonly onSave: (physicianData: DoctorFormData) => void; // Callback that accepts the inputs defined in this page
+  readonly onSave: (applicationData: any) => void;
```

Comment:
Is there a reason the type of this prop was changed? I think we should probably keep it as `physicianData: DoctorFormData` unless i'm missing something

Reply:
hmm I guess no major rationale, I just didn't see a reason for it to be defined this way so I tried to keep it in line with how the rest of the edit modals were written

Reply:
Okay, I think if we could at least bring back the type to be `DoctorFormData` since thats always the type that will be passed to the onSave so it just makes the code a bit safer.